### DISCUSSION TITLE
How to test Theia electron app using Spectron ?

### DISCUSSION THREAD

**Discussion by kingkan143** [eclipse-theia/theia#8598] [Q&A]
Need to test Theia Electron app as Integrated system test.

**Comment by vince-fugnitto** [comment]
@kingkan143 we currently do not test our example-electron application with Spectron.
If you're going to add Spectron for your own application then the best place to start would be their [dcoumentation](https://www.electronjs.org/spectron) (I'm not sure how much the community can help as it was never added before).
